src/pages/index.js

```diff
@@ -130,9 +130,9 @@ function renderCard(cardData) {
             cardElement.removeCardElement(res._id);
             cardDeletePopup.close();
           })
-          .catch(console.err)
+          .catch(console.error)
           .finally(() => {
-            cardDeletePopup.setLoading(false, "Yes");
+            cardDeletePopup.setLoading(false);
```

Comment:
See here https://disk.yandex.ru/i/SDBqv7PlE9pHvw . After the 1st deleting the button text is lost


So, in `PopupWithConfirm` you need to correct 

```js
setLoading(isLoading, loadingText = "Deleting...") {

```


And `this._submitButtonText = this._submitButton.textContent;` should be in the constructor